pkg/cmd/server/api/types.go

```diff
@@ -8,8 +8,8 @@ import (
 
 const (
 	FeatureBuilder    = `Builder`
-	FeatureS2I        = `S2I Builder`
-	FeatureWebConsole = `Web Console`
+	FeatureS2I        = `S2IBuilder`
```

Comment:
> what is driving changing these?

@smarterclayton made comments in the previous pull after I merged it.  He said no spaces.